test/cmp_ctx_test.c

```diff
@@ -118,6 +118,7 @@ static int msg_total_size_log_cb(const char *func, const char *file, int line,
                                  OSSL_CMP_severity level, const char *msg)
 {
     msg_total_size += strlen(msg);
+    printf("total=%d len=%ld msg=\"%s\"\n", msg_total_size, strlen(msg), msg);
```

Comment:
Why do we have a printf here?

Reply:
The output is very helpful for debugging in case the test fails.
I've re-phrased the statement to use `BIO_printf(bio_err ....)`.